packages/office-ui-fabric-react/src/components/MarqueeSelection/MarqueeSelection.base.tsx

```diff
@@ -216,22 +217,52 @@ export class MarqueeSelectionBase extends BaseComponent<IMarqueeSelectionProps,
 
           this._preservedIndicies = selection && selection.getSelectedIndices && selection.getSelectedIndices();
         }
+
         // We need to constrain the current point to the rootRect boundaries.
-        const constrainedPoint = this.props.isDraggingConstrainedToRoot
```

Comment:
It seems like `isDraggingConstrainedToRoot` might not be respected any more. Did this get dropped?

Reply:
It turns out that isDraggingConstrainedToRoot was working previously.  I think that the person who made the change on our end was not aware of this prop, so they rewrote / broke the behavior.  Sorry I did not catch this, and thank your for noticing.

Reply:
So, I have removed our changes related to constraining the drag rect.  It meets our needs to just set isDragginConstrainedToRoot to true.